[2024-05-24T03:38:27.681+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:38:26.548040+00:00 [queued]>
[2024-05-24T03:38:27.694+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:38:26.548040+00:00 [queued]>
[2024-05-24T03:38:27.695+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-05-24T03:38:27.710+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): execute_teste_py> on 2024-05-24 03:38:26.548040+00:00
[2024-05-24T03:38:27.717+0000] {standard_task_runner.py:57} INFO - Started process 177 to run task
[2024-05-24T03:38:27.721+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'minio_python_task', 'execute_teste_py', 'manual__2024-05-24T03:38:26.548040+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/dag_teste/teste_***.py', '--cfg-path', '/tmp/tmp_52mkpbk']
[2024-05-24T03:38:27.723+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask execute_teste_py
[2024-05-24T03:38:27.786+0000] {task_command.py:415} INFO - Running <TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:38:26.548040+00:00 [running]> on host 996d4eb998ba
[2024-05-24T03:38:27.923+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_python_task' AIRFLOW_CTX_TASK_ID='execute_teste_py' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T03:38:26.548040+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-24T03:38:26.548040+00:00'
[2024-05-24T03:38:27.932+0000] {logging_mixin.py:151} INFO - Iniciando instalação do PySpark...
[2024-05-24T03:38:27.933+0000] {logging_mixin.py:151} INFO - O diretório de trabalho atual é: /opt/***
[2024-05-24T03:38:27.933+0000] {logging_mixin.py:151} INFO - Iniciando a instalação do Java a partir do repositório do GitHub...
[2024-05-24T03:38:27.936+0000] {logging_mixin.py:151} INFO - Ocorreu um erro inesperado durante a instalação do Java a partir do repositório do GitHub: [Errno 2] No such file or directory: '/java:/opt/***/java'
[2024-05-24T03:38:27.937+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-24T03:38:27.956+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=minio_python_task, task_id=execute_teste_py, execution_date=20240524T033826, start_date=20240524T033827, end_date=20240524T033827
[2024-05-24T03:38:28.023+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-24T03:38:28.047+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
